proposals-CG-WG.md

```diff
@@ -38,42 +38,45 @@ The purpose of Stage 1 proposals is to explore the Solution Space and pick a pre
-    * [ ] Confidence of developer demand and fitness for purpose (e.g. a developer that needs this proposal)
+    * [ ] An [explainer](https://tag.w3.org/explainers/).
+    * [ ] Documentation of alternatives and trade-offs considered.
+    * [ ] Optionally, a draft specification text (or detailed examples/code samples if needed for clarity).
+    * [ ] Early implementation experience, including prototypes or trials.
+    * [ ] Evidence of stakeholder (e.g. web developers) demand and use-case alignment.
+
   * What's asked of the **Working Group**?
-    * [ ] Working Group consensus to adopt the proposal as the basis for their work as a [Working Draft](https://www.w3.org/policies/process/#RecsWD).
-    * [ ] Working Group identification of the list of (seemingly resolvable) issues that have to be addressed before [Stage 3](#stage-3).
-    * [ ] An [Editor's Draft](https://www.w3.org/policies/process/#editors-draft) can be used to get PRs merged between [Working Draft](https://www.w3.org/policies/process/#RecsWD) revisions.
+    * [ ] Working Group consensus to adopt the proposal as the basis for their work.
+    * [ ] A clear list of blocking issues to be addressed before advancing to Stage 3.
+    * [ ] A completed Working Draft for further iteration.
+    * [ ] Approval to dedicate the WG's time to Review spec PRs to merge the proposal into the [Working Draft](https://www.w3.org/policies/process/#RecsWD).
+
   
 # Stage 3: Implementation
 
 > NOTE: We don't yet have enough experience as we get towards Stage 3 and 4, so they aren't fully baked yet. It is possible that these
-> will change a lot as we learn, including potentially entirely replacing them with references to the W3C Process.
+> will change a lot as we learn, including potentially entirely replacing them with references to the W3C Process. 
 
-The purpose of Stage 3 Proposals is to increase implementation and deployment confidence in order to produce a [Candidate Recommendation](https://www.w3.org/policies/process/#RecsCR).
+The purpose of Stage 3 Proposals is to increase implementation and deployment confidence in order to produce a [Candidate Recommendation](https://www.w3.org/policies/process/#RecsCR). The spec should be considered finished, pending editorial nit review. However, since multiple implementations are expected to be produced during this stage, it is possible there will be further changes to normative content based on feedback from those implementors.
  
   * What's needed from champions?
-    * [ ] Proposal fully merged into the [Working Draft](https://www.w3.org/policies/process/#RecsWD)
-    * [ ] Web Platform Tests
-    * [ ] Further implementation experience
+    * [ ] Proposal fully merged into the [Working Draft](https://www.w3.org/policies/process/#RecsWD).
+    * [ ] Web Platform Tests are available.
+    * [ ] At least two independent implementers and no unresolved objections (where resolution may include an agreement not to address the issue).
```

Comment:
This requirement seems a bit off compared with the previous text `since multiple implementations are expected to be produced during this stage`. So are multiple implementors needed to reach stage 3 or could they possibly occur during stage 3? Note that in WHATWG it is two implementers supporting which is different from having actually implemented. https://whatwg.org/stages#stage3

Reply:
Agreed. Stage 3 is ready for implementation; so rephrasing to supporting makes sense.

Reply:
Your agreement does not seem reflected in the text